src/main/java/me/contaria/seedqueue/mixin/compat/atum/OptionsScreenMixin.java

```diff
@@ -0,0 +1,72 @@
+package me.contaria.seedqueue.mixin.compat.atum;
+
+import com.bawnorton.mixinsquared.TargetHandler;
+import com.llamalad7.mixinextras.injector.wrapoperation.Operation;
+import com.llamalad7.mixinextras.injector.wrapoperation.WrapOperation;
+import me.contaria.seedqueue.SeedQueue;
+import net.minecraft.client.gui.screen.Screen;
+import net.minecraft.client.gui.screen.options.OptionsScreen;
+import net.minecraft.client.gui.widget.ButtonWidget;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableText;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(value = OptionsScreen.class, priority = 1500)
+public abstract class OptionsScreenMixin extends Screen {
+    @Unique
+    private ButtonWidget atumButton;
+
+    protected OptionsScreenMixin(Text title) {
+        super(title);
+    }
+
+    @Unique
+    private boolean shouldClearQueue() {
+        return Screen.hasShiftDown() && SeedQueue.isActive();
+    }
```

Comment:
Codestyle: i prefer added unique methods to be at the bottom of my mixin, below the actual injections